packages/app/src/app/pages/Dashboard/Components/EmptyScreen/index.tsx

```diff
@@ -29,12 +29,12 @@ export const EmptyScreen: React.FC<EmptyScreenProps> = ({
     );
   }
 
-  if (page === 'archive') {
+  if (page === 'deleted') {
     return (
       <Stack justify="center" align="center" marginTop={120}>
         <Text variant="muted">
-          There are no archived sandboxes yet! Drag sandboxes or templates to
-          this page to archive them.
+          There are no deleted sandboxes yet! Drag sandboxes or templates to
```

Comment:
Should we keep saying `drag` or change it to `move`, since we can also move them with the context menu?

Reply:
I think it's ok to use the word drag so people aware that they can do that. the context menu is pretty explicit, but the drag capabilities might be hidden at first